go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/cilium/customvet v0.0.0-20201209211516-9852765c1ac4
 	github.com/cilium/deepequal-gen v0.0.0-20200406125435-ad6a9003139e
-	github.com/cilium/ebpf v0.5.1-0.20210421150058-a4ee356536f3
+	github.com/cilium/ebpf v0.6.2-0.20210621092238-2d8d7b4c6e10
```

Comment:
>  [wip] vendor: Update cilium/ebpf
>
> This bumps to 2d8d7b4c6e103a8c273bf3aaa86c417e316ee72d which includes
support for BPF_F_MMAPABLE.
>
> TODO: Update when BPF_F_INNER_MAP support is merged

Can you please update the subject and log message for the commit related to this change? (Should it also be re-ordered to appear before the commits using `BPF_F_INNER_MAP`?)

Reply:
Yep, I posted this PR with this commit still pending because I wanted to get feedback on the rest of it, without blocking on such a minor part.

Reply:
Good catch on the `BPF_F_INNER_MAP` being used before it has support in ciilum/ebpf.